ISISComputingGroup/ibex_gui
# Ticket1358: Refactor InstrumentVariables

### Description of work

The intended use of the instrument variables class has changed after the switching factory refactor. This refactor updates places in the GUI to reflect the change in use.

### To test 

See https://github.com/ISISComputingGroup/IBEX/issues/1358

Ensure that PVs in the GUI work as previously

---

#### Code Review


### Functional Tests


### Final Steps
